azavea/pfb-network-connectivity
# Feature/kak/org admin group#389

## Overview

Add user role for organization administrators. Can create users within their org and edit non-admin users in org; otherwise have viewer permissions.

### Demo

In UI, org admin has access to manage users but not organizations. 'Edit' button hidden by full admins from org admins, as they don't have access to modify full admins.

![image](https://cloud.githubusercontent.com/assets/960264/25861155/d9ed25a8-34b1-11e7-972e-e734e3bf4cbb.png)


### Notes

Also removes editor and uploader roles (closes #405.) Fixes up error message display in the UI for user management, but it looks like UI error handling generally could use some cleanup and additions; analysis job creation, for example, does not display correctly. Created #406 for additional UI error handling cleanup.


## Testing Instructions

 * Run django migrations
 * Any existing uploaders or editors should now have viewer role
 * Check browsing admin angular app UI as org admin user
 * Org admin user should be able to create and manage non-admin users in their org
 * Org admin user should not be able to create a full admin or make existing user a full admin
 * Org admin users should otherwise have viewer access
 * Uploader and editor roles should not longer be displayed in drop-down options

Closes #389. Closes #405.
